helmfile.d/jenkins-infra.yaml

```diff
@@ -0,0 +1,17 @@
+helmDefaults:
+  atomic: true
+  force: true
+  timeout: 600
+  wait: true
+
+releases:
+    - name: default-jenkins-infra
+      chart: ../charts/jenkins
+      namespace: jenkins-infra
+      values:
+        - "../config/default/jenkins-infra.yaml"
+      secrets:
+        - "../secrets/config/jenkins-infra/jenkins.yaml"
+      set:
+        - name: namespace
```

Comment:
I dont think you need namespace above and here

Reply:
Yes I do need to configure that namespace to tell to the k8s plugin where to spinup pods, I just need to update it to the right place, thanks